testing_application/app.py

```diff
@@ -0,0 +1,82 @@
+from typing import TypedDict
+
+from robyn import Robyn, OpenAPI, Request, SubRouter
+from robyn.openapi import OpenAPIInfo, Contact, License, ExternalDocumentation, Components
+
+app: Robyn = Robyn(
+    file_object=__file__,
+    openapi=OpenAPI(
+        info=OpenAPIInfo(
+            title="Sample App",
+            description=" This is a sample server application.",
+            termsOfService=" https://example.com/terms/",
+            version=" 1.2.2",
+            contact=Contact(
+                name="API Support",
+                url="https://www.example.com/support",
+                email="support@example.com",
+            ),
+            license=License(
+                name="BSD2.0",
+                url="https://opensource.org/license/bsd-2-clause",
+            ),
+            externalDocs=ExternalDocumentation(description="Find more info here", url="https://example.com/"),
+            components=Components(),
+        ),
+    ),
+)
+
+
+@app.get("/")
+async def welcome():
+    """welcome endpoint"""
+    return "hi"
+
+
+class GetRequestParams(TypedDict):
+    appointment_id: str
+    year: int
+
+
+@app.get("/api/v1/:name", openapi_tags=["Name"])
+async def get(r: Request, query_params=GetRequestParams):
+    """Get Name by ID"""
+    return r.query_params
```

Comment:
where are the query params in this route?

Reply:
what?

Reply:
@VishnuSanal `:/name` is a path parameter. You are returning query parameter.

Reply:
@VishnuSanal , where is the path param being used?

Reply:
> @VishnuSanal , where is the path param being used?

um... it is not "used", but that will show up in the swagger docs as a field.

![image](https://github.com/user-attachments/assets/d9efea34-75b7-40df-855c-5cf226fd51cb)

should I remove it?

Reply:
@VishnuSanal , yes please. Let us keep the test routes specific to their function.

Reply:
@sansyrox done, PTAL. :)